corehq/apps/cloudcare/static/cloudcare/js/form_entry/entries.js

```diff
@@ -150,6 +170,10 @@ hqDefine("cloudcare/js/form_entry/entries", function () {
             if (match) {
                 var receiveTopic = match[1];
                 var receiveTopicField = match[2];
+                if (receiveTopicField === constants.RECEIVER_FIELD_INDEXED) {
+                    var ixMatch = question.ix().match(/_(\d)/);
```

Comment:
should that be `/_(\d+)/` in case we get more than 10 files?

Reply:
Yep, good call. https://github.com/dimagi/commcare-hq/pull/34137/commits/903a80f764b017e509fc25dc2d4764854163c072